go/test/endtoend/vtgate/queries/aggregation/schema.sql

```diff
@@ -105,3 +105,25 @@ CREATE TABLE bet_logs (
     game_id bigint,
     PRIMARY KEY (id)
 ) ENGINE InnoDB;
+
+create table example (
+    id bigint unsigned not null,
+    foo enum('a','b') not null,
+    primary key (id)
+) Engine = InnoDB;
+
+-- Table used to exercise unknown enum value fallback hashing. The vschema for this
+-- table deliberately omits the "c" value so we can insert known and unknown enums.
+create table example_enum_unknown (
+    id bigint unsigned not null,
+    foo enum('a','b','c') not null,
+    primary key (id)
+) Engine = InnoDB;
+
+-- Table used to exercise unknown set value fallback hashing. The vschema for this
+-- table deliberately omits the "c" value so we can insert known and unknown sets.
+create table example_set_unknown (
+    id bigint unsigned not null,
+    foo set('a','b','c') not null,
+    primary key (id)
+) Engine = InnoDB;
```

Comment:
This seems wrong. The comment says that the tables omit the value 'c' so that the test can insert 'c' and we can check the unknown value insertion, but the table definition includes 'c' in both the enum and set, so we aren't testing what we want.

Reply:
Ignore this comment, everythign is fine.